### PR TITLE
Readme cleanup, update to release notes, ʻlight modeʻ screenshots

### PR DESCRIPTION
**Pull request by wdpk**
update the readme to have more of a present date, clean up various small grammar and formatting throughout the page.

clarify that the docs donʻt have release notes written in the same way as the rest of the project. include information about what makes it to the git commit messages to encourage people to write good PRs.

add note that screenshots should be taken in light mode

Notes and edits welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,19 +166,21 @@ If you want to use code in your documentation, that is easy too.
...
 
 ### Pull Requests
 
-When submitting a pull request, please make sure that your contributions are complete, I am checking this repo regularly so if I see a open PR the likelihood of me pulling it and merging it are high.
+When submitting a pull request, please make sure that your contributions are complete. I am checking this repo regularly, so if I see an open PR, the likelihood of me pulling it and merging it is high.
 
-Documentation submitted will be proof read and amended before merging, please don't take this personally if any of your documentation gets amended, we just want to make sure all documents are ready to go before merging them into main.
+Documentation submitted will be proofread and amended before merging. Please don't take this personally if any of your documentation gets amended; we just want to make sure all documents are ready to go before merging them into main.
```
Comment (matt-fidd) [inline README.md:179]: really minor nit, the trunk branch is called `master` in this repo

Comment (wdpk) [reply README.md]: good catch. but maybe that itself should be revisited.

File: README.md [hunk 2]
```diff
@@ -220,4 +222,4 @@ If you are using GitHub pages for hosting, this command is a convenient way to b
 
 ### Writing Good Release Notes
 
-This is detailed in the [contributing](https://actualbudget.org/docs/contributing/#writing-good-release-notes) section of the documentation.
+The Documentation site is not versioned in the same manner as the rest of the project. The Documentation website is more of a living document. Merges into `master` will have the title and PR# as commit message, with the PR description as extended git commit description.
```
Comment (youngcw) [inline README.md:225]: Can we still have that release notes link here?  It has good info in it

File: README.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+Actual Budget can be installed on many different platforms. However, at this time, the official Actual Budget documentation only supports the following methods:
 
-- Local Installation (on your own machine)
-- Fly.io
-- PikaPods
-- Docker
+* Local Installation (on your own machine)
+* Fly.io
+* PikaPods
+* Docker
```
[the remark this answers is not in the document]
Comment (wdpk) [reply README.md:18]: took a whack at clarifying between local methods. i think that client is technically a third way, but with desktop apps, people are going to gravitate towards desktop-app or full-blown-server i think

**Pull request merged**